ui/utils/custom-filter.js

```diff
@@ -0,0 +1,174 @@
+import React, { useState } from 'react';
+import { Popper } from '@mui/material';
+import FilterIcon from '../assets/icons/filter';
+import {
+  Button,
+  Card,
+  ClickAwayListener,
+  InputLabel,
+  MenuItem,
+  Select,
+  Tooltip,
+  IconButton,
+} from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = (theme) => ({
+  epaper: {
+    '&: .MuiPaper-root': {
+      background: theme.palette.secondary.headerColor,
+      color: theme.palette.secondary.textMain,
+    },
+    padding: '1rem',
+    boxShadow: `0px 4px 8px ${
+      theme.palette.type === 'light'
+        ? theme.palette.secondary.disabledIcon
+        : '0px 4px 5px 0px rgba(0,0,0,0.14)'
+    }`,
+  },
+  icon: {
+    color: theme.palette.secondary.iconMain,
+  },
+});
```

Comment:
Styled component would be nice

Reply:
styled components are messing with the theme currently , so when we work on v5 these all can be converted